src/js/GlobalSettings/Hardware/Hardware.tsx

```diff
@@ -0,0 +1,24 @@
+import { isMobile } from "react-device-detect";
+import { Hardware } from "../../types/Types";
+
+export const DEFAULT_HARDWARE_TYPE = "windows";
+export const HARDWARE_TYPES: Hardware[] = [
+  "iphone",
+  "android",
+  "windows",
+  "mac",
+];
+
+export function isPhoneHardware(): boolean {
+  return isMobile;
+}
```

Comment:
I think better to delete it and just use the value from lib)

Reply:
Done